tests/unit/rdf/test_resource_serialisation.py

```diff
@@ -314,6 +315,175 @@ class A(NewResource):
+    """Testing unordered list in a markdown isn't detected as html."""
+    example_html_in_markdown = """
+    - First item
+    - Second item
+    - Third item
+    """
+    map_str_to_markdown(example_html_in_markdown)
+    captured = capsys.readouterr()
+    assert captured.out + captured.err == ""
+
+def test_ordered_list_detected_as_html(capsys):
+    """Testing ordered list in a markdown isn't detected as html."""
+    example_html_in_markdown = """
+    1. First item
+    2. Second item
+    3. Third item
+    """
+    map_str_to_markdown(example_html_in_markdown)
+    captured = capsys.readouterr()
+    assert captured.out + captured.err == ""
+
+def test_image_detected_as_html(capsys):
+    """Testing image in a markdown isn't detected as html."""
+    example_html_in_markdown = """
+    ![alt text](image.jpg)
+    """
+    map_str_to_markdown(example_html_in_markdown)
+    captured = capsys.readouterr()
+    assert captured.out + captured.err == ""
+
+def test_blockquote_tag_detected_as_html(capsys):
+    """Testing blockquote in a markdown isn't detected as html."""
+    example_html_in_markdown = """
+    > blockquote
+    """
+    map_str_to_markdown(example_html_in_markdown)
+    captured = capsys.readouterr()
+    assert captured.out + captured.err == ""
+
+def test_headings_detected_as_html(capsys):
+    """Testing headings in a markdown isn't detected as html."""
+    example_html_in_markdown = """
+    # H1
+    ## H2
+    ### H3
+     """
+    map_str_to_markdown(example_html_in_markdown)
+    captured = capsys.readouterr()
+    assert captured.out + captured.err == ""
+
+def test_bold_text_is_detected_as_html(capsys):
+    """Testing bold text in a markdown isn't detected as html."""
+    example_html_in_markdown = """
+    **bold text**
+    """
+    map_str_to_markdown(example_html_in_markdown)
+    captured = capsys.readouterr()
+    assert captured.out + captured.err == ""
+
+def test_none_tag_detected_as_html(capsys):
+    """Testing variations of greater than and less than signs, don't get identified as html"""
+    example_html_in_markdown = """
+    # This is some example markdown without any html
+    but does contain these < > and these >< and these > < and this <> 
+    """
+    map_str_to_markdown(example_html_in_markdown)
+    captured = capsys.readouterr()
+    assert captured.out + captured.err == ""
+
+def test_algebraic_expressions_with_spaces__is_detected_as_html(capsys):
+    """Testing algebraic expressions that have spaces between the 
+    greater than and less than symbols don't get identified as html"""
+    example_html_in_markdown = """
+    x > y,
+    x < y,
+    x < z < y,
+    x > y > z
+    """
+    map_str_to_markdown(example_html_in_markdown)
+    captured = capsys.readouterr()
+    assert captured.out + captured.err == ""
+
+def test_algebraic_expressions_without_spaces_is_detected_as_html(capsys):
+    """Testing algebraic expressions get identified as html even though
```

Comment:
Just make it clear in the comment that  you're including this test as a demonstration - not as desired behaviour.